backend/src/auth/config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * firebase SDK details can be found here:
+ * https://firebase.google.com/docs/admin/setup
+ */
+
+const admin = require('firebase-admin')
+const private_key_id = process.env.FIREBASE_PRIVATE_KEY_ID // eslint-disable-line
```

Comment:
why are we doing this again? There's an example on the firebase website which just imports the json with all the credentials in there.